src/pages/home/report/ReportActionsView.js

```diff
@@ -155,7 +155,7 @@ class ReportActionsView extends React.Component {
         const previousLastSequenceNumber = lodashGet(lastItem(prevProps.reportActions), 'sequenceNumber');
         const currentLastSequenceNumber = lodashGet(lastItem(this.props.reportActions), 'sequenceNumber');
         const shouldRecordMaxAction = Visibility.isVisible()
-            && !(this.props.isDrawerOpen && this.props.isSmallScreenWidth);
+            && (!this.props.isSmallScreenWidth || !this.props.isDrawerOpen);
```

Comment:
Can we please add a comment here and list out every situations where we will record the max action?